graylog2-web-interface/src/components/common/EntityDataTable/EntityDataTable.tsx

```diff
@@ -98,12 +98,13 @@ const filterVisibleColumns = (
 
 const useElementsWidths = <Entity extends EntityBase>({
   columns,
-  columnRenderers,
+  columnRenderersByAttribute,
   displayBulkSelectCol,
 }: {
   columns: Array<Column>,
-  columnRenderers: ColumnRenderers<Entity>,
+  columnRenderersByAttribute: ColumnRenderersByAttribute<Entity>,
   displayBulkSelectCol: boolean
+  entityAttributesAreCamelCase: boolean
```

Comment:
The name of this prop sounds like our attributes already in camel case. But then, if this attr is `true`, we are converting columns to camel case. Maybe make sense to rename it to smth like `shouldEntityAttributesBeCamelCase` or `setEntityAttributesToCamelCase`

And, just another idea, to avoid props drilling, we can create a kind of `<EntityDataTable />` Context and pick props like `entityAttributesAreCamelCase` in, f.e. `<TableCell />`, from the Context. But that is up to you, currently hard to say would we need a lot of props from  `<EntityDataTable />` in other parts of the table

Reply:
`The name of this prop sounds like our attributes already in camel case` - When the attribute is true we actually expect, that the entity attributes are camel case. The logic behind it is, that the column / attribute ids in the column definitions / metadata are always snake case. If `entityAttributesAreCamelCase` is true we transform the column ids to camel case to be able to make a connection with the entity attributes.

Reply:
I like the idea with the contexts.

Reply:
I improved the props description for `entityAttributesAreCamelCase`. I hope it makes it more clear.